9_report/src/fig_input_data.R

```diff
@@ -18,27 +18,65 @@ fig_input_data <- function(fig_ind, input_example_yml, preds_ind, remake_file, c
               nwm_forecast_ind = '2_munge/out/agg_nwm_long1.rds.ind',
               remake_file = '3_forecast.yml')
 
+  flux_nwis <- eList$Sample %>%
+    dplyr::filter(Uncen == 1) %>% # only two data points that are censored (removing for now)
```

Comment:
Pro of this approach: simpler plot, fewer distractions.

Con: people who care about censored data will look for it and be worried when they don't see any nod to them. Then again, since I'll be standing at the poster, I can always clarify.

I think I'm good with the decision to filter, at least for now.

Reply:
I'll put them back in as grey and see how it looks.